### ISSUE TITLE
Linux Tarball Install test tracking

### ISSUE THREAD

**Issue by jberkus** [voltraco/beta#9]
This issue is to track which versions of Linux the tarball install has been verified to work on.

Linux tarball install verified to work on Fedora 23, x86_64.

Minor issues:
- need README or online install intructions or both, directing user to put unzipped tarball in /usr/share
- tarball directory is 'voltra64', but application wants 'voltra'

**Comment by heapwolf** [comment]
Proposing this README.md for packaging with linux tarballs...

```
     _  __       ____  ___   ___
  | /  /  /  /    /   /__/  /__/
  |/  /__/  /__  /   /  \  /  /

  Music Player and Store
  Voltra Co. (https://voltra.co)

  # INSTALL
  We recommend extracting the contents of the tarball to `/user/share/voltra`

  # HELP!
  Create issues on `https://github.com/voltraco/beta` otherwise message us
  on `https://twitter.com/voltraco`. You can also email us at `hello@voltra.co`.
  You can chat with us by joining our slack community, at `https://voltra.co/slack`.
```

**Comment by rkyleg** [comment]
I tried this on Linux Mint 18. Voltra would not load (stuck on splash screen) when I extracted the tarball into HOME/src. Only after I moved the voltra64 folder to /usr/share and renamed to voltra did the application load. I think these two actions should be called out and stressed for a successful Linux install, at least for Ubuntu/Linux Mint.

**Comment by heapwolf** [comment]
This should all be fixed with the new build options for linux. Please feel free to open a new issue with as many specifics as possible if you find it's still not working for you.
